Example output:

````
"FAILURES": {
            "ID": "WCS_UNHANDLED_ERROR",
            "STACK": "com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $\n\tat com.google.gson.Gson.fromJson(Gson.java:900)\n\tat com.google.gson.Gson.fromJson(Gson.java:853)\n\tat com.google.gson.Gson.fromJson(Gson.java:802)\n\tat com.hpe.caf.worker.entityextract.core.EntityExtractOperationBuilder.getEntities(EntityExtractOperationBuilder.java:170)\n\tat com.hpe.caf.worker.entityextract.core.EntityExtractOperationBuilder.build(EntityExtractOperationBuilder.java:55)\n\tat com.hpe.caf.worker.entityextract.EntityExtractWorker.processDocument(EntityExtractWorker.java:86)\n\tat com.hpe.caf.worker.document.DocumentMessageProcessor.processDocument(DocumentMessageProcessor.java:176)\n\tat com.hpe.caf.worker.document.DocumentMessageProcessor.processDocumentHierarchy(DocumentMessageProcessor.java:154)\n\tat com.hpe.caf.worker.document.DocumentMessageProcessor.processTask(DocumentMessageProcessor.java:139)\n\tat com.hpe.caf.worker.document.DocumentMessageProcessor.doWork(DocumentMessageProcessor.java:82)\n\tat com.hpe.caf.worker.core.StreamingWorkerWrapper.run(StreamingWorkerWrapper.java:55)\n\tat java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)\n\tat java.util.concurrent.FutureTask.run(FutureTask.java:266)\n\tat java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)\n\tat java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)\n\tat java.lang.Thread.run(Thread.java:748)\nCaused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $\n\tat com.google.gson.stream.JsonReader.beginObject(JsonReader.java:385)\n\tat com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:183)\n\tat com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145)\n\tat com.google.gson.Gson.fromJson(Gson.java:888)\n\t... 15 more\n",
            "WORKFLOW_ACTION": "entity_extract",
            "COMPONENT": "worker-entityextract 4.3.0-SNAPSHOT",
            "WORKFLOW_NAME": "ingestion-workflow",
            "MESSAGE": "java.lang.IllegalStateException: Expected BEGIN_OBJECT but was STRING at line 1 column 1 path $",
            "DATE": "2020-02-03T21:39:49.978Z",
            "CORRELATION_ID": "eb8d7532-5cf8-41b0-aafd-6b0043ca62d0",
            "AJP_JOB_RUN_ID": "42",
            "AJP_WORK_UNIT_ID": "43"
          },
````

--- a/worker-workflow/src/main/resources/workflow-control.js
+++ b/worker-workflow/src/main/resources/workflow-control.js
@@ -121,7 +121,7 @@ function setWorkerVersion(document) {
 function onError(errorEventObj) {
     var rootDoc = errorEventObj.rootDocument;
     var message = errorEventObj.error.getMessage();
-    rootDoc.getFailures().add("WCS_UNHANDLED_ERROR", message, errorEventObj.error);
+    rootDoc.failures.add("UNHANDLED_ERROR", message, errorEventObj.error);
     errorEventObj.handled = true;
     traverseDocumentForFailures(rootDoc);
     routeTask(errorEventObj.rootDocument);

